lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -523,7 +560,9 @@ _LoadTexture(const std::string& path, bool& isColorSpaceSRGB, MFloatArray& uvSca
         isColorSpaceSRGB = image->IsColorSpaceSRGB();
         texture = textureMgr->acquireTexture(path.c_str(), desc, spec.data);
         break;
-    default: break;
+    default:
+        TF_WARN("VP2 renderer delegate: unsuported pixel format: %d.", (int)spec.format);
```

Comment:
There is a typo: `unsuported` -> `unsupported`

Maybe you can print the file path as well for easier lookup/debugging?